md/installation_guide.md

```diff
@@ -0,0 +1,238 @@
+* i: the port of the deployed ICI application (default '8082') 
+* e: the port of the deployed elasticsearch (default '9200')    
+
+:::note
+use `./bonita-ici --help` displays all options
+:::
+
+It can be stopped using `bonita-ici stop`
+
+## Production mode 
+The deliverable is composed of one standalone application, two Bonita Living Applications, and an installer that will
+help you install the backend and deploy the two living applications in your current instance of the Bonita platform.
+
+```
+bonita-ici-<VERSION>.zip
+    |---- ici-application-<VERSION>.zip           // Standalone application for A.I.
+    |---- la-configuration-<VERSION>.zip          // Bonita L.A. for I.C.I. configuration
+    |---- la-operations-management-<VERSION>.zip  // Bonita L.A. for Operations Management
+    |---- la.properties                           // properties used by L.A. deployer utility
+    |---- installation-guide.md                   // this file
+    |---- bin/bonita-ici                          // an utility that start the dockers and deploy L.A.s
+    |---- bin/bonita-ici.bat                      // an utility that start the dockers and deploy L.A.s (Windows)
+    |---- jdbc_drivers/                           // the folder where to put the jdbc driver of the Bonita platform database
+```
+
+
+### Elasticsearch
+
+ICI server requires elasticsearch. Supported version is 6.2.3 or above, in cluster or single node mode. 
+
+For more information, please refer to the [official installation guide](https://www.elastic.co/downloads/elasticsearch)
+
+
+### ICI server
+
+1. Copy ici-application-<VERSION>.zip` to the host. 
+2. Unzip it in the directory of your choice. 
+
+We will refer to the extracted directory as `ICI_APPLICATION_HOME`.
+
+#### Configuration
+
+Edit `$ICI_APPLICATION_HOME/application.properties` to configure application.
+Mandatory parameters are:
+
+In Elastic search configuration
+```
+bonita.elasticsearch.hosts              // http url(s) of installed Elasticsearch 
+
+# when elastic search endpoint are protected by authentication,
+# rest client basic authentication can be enabled by adding properties as follow
+#bonita.elasticsearch.username=admin
+#bonita.elasticsearch.password=changeme
+```
+
+In Bonita database configuration
+```
+bonita.datasource.url                   // url of Bonita platform database
+bonita.datasource.username              // username used to connect to Bonita database (can be a read-only user)
+bonita.datasource.password              // password associated to the user used to connect to Bonita database
+```
+
+Oracle only: copy Jdbc driver in `$ICI_APPLICATION_HOME/lib` (drivers for PostgreSQL, MySQL and MS SqlServer are already provided).
+
+#### Advanced configuration
+
+Other parameters are available in `$ICI_APPLICATION_HOME/application.properties`.  
+We recommend you to read this file and change other parameters if needed.
+
+#### Advanced polling profile mode (Oracle only)
```

Comment:
polling profile? Is this the standard terminology?

Reply:
for instance, yes